src/static/riot/competitions/tile/front_page_competitions.tag

```diff
@@ -0,0 +1,93 @@
+<front-page-competitions>
+    <div hide="{CODALAB.state.user.logged_in}" class="vert-wrap">
```

Comment:
It looks like the only difference between this div and its counterpart is the overall class `logged_in`?
can save a lot of duplicate code w/
```html
<div class="vert-wrap {logged-in: CODALAB.state.user.logged_in}">
```
Then you only have to have this whole div once, and if the user is logged in, add the `logged-in` class

Reply:
Nice! That got rid of a bunch of repetition on this tag